SEED-platform/seed
# Hand-merge multiple records

#### What's this PR do?
Instead of going to the matching page, where it only allows you to match against previously-unmatched records from data import files, even if those records don't appear in the inventory list view, you can now use the checkboxes on the inventory list page to preview and merge multiple records at once.

#### How should this be manually tested?
- Go to the inventory list page, check two or more checkboxes, click the `Actions` dropdown, click `Merge Selected`, and then reorder the records in the preview modal by dragging to confirm the intended merge result.
- Merged records can be unmerged from the inventory detail view.
- Paired properties/taxlots can be unpaired from the inventory detail view.
- Please also monkey test the inventory list/detail/settings views and quite a bit of refactoring has gone on under the hood (for instance, inventory detail hyperlinks no longer have a property/taxlot id and a cycle id, they simply use a propertyView or taxlotView id.

#### What are the relevant tickets?
#1385 

#### Screenshots (if appropriate)
1. Merge Multiple Selected Properties
![1 merge multiple selected properties](https://user-images.githubusercontent.com/411466/35045360-d705f110-fb50-11e7-9fdb-28d3a7dda81b.png)
2. Reorder and Preview Merge Result
![2 reorder and preview merge result](https://user-images.githubusercontent.com/411466/35045365-da45981c-fb50-11e7-9a6d-b469fd2be39a.png)
3. Unpair from the Detail view
![3 unpair from the detail view](https://user-images.githubusercontent.com/411466/35045367-dcd76a6a-fb50-11e7-8a0b-fef5058ec545.png)
4. Unmerge Last from the Detail view
![4 unmerge last](https://user-images.githubusercontent.com/411466/35045382-e5746d94-fb50-11e7-99bb-47b114003767.png)
